boards/renesas/ek_ra4m1/ek_ra4m1_defconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=48000000
```

Comment:
shouldn't this be set by SoC?

Reply:
I believe it is more reasonable to set this value on board, as it will depend on the actual clock supply available on the boards.

Reply:
@gmarull

Reply:
this likely means you need to take this from DT?

Reply:
sorry for late response,
I moved it into SoC and took it from DT, thank you